spire-flex/tests/agentHealthCheckBindPort.yaml

```diff
@@ -22,6 +22,11 @@ tests:
             }
             
             plugins {
+              KeyManager "disk" {
```

Comment:
I'm not 100% sure about the KeyManager "disk" being the default.  It will prevent node-reattesation, so that's the rationale.  If we don't care about avoiding node-reattesation, we could pivot to "memory".

Reply:
If the pod gets recreated on the same node, then it should still honor its old certs I think, so disk makes sense to me as a default.

Reply:
Ok.  Moving forward with the KeyManager "disk" as the correct default.